packages/communication-react/review/communication-react.api.md

```diff
@@ -1254,20 +1254,20 @@ export interface ControlBarButtonProps extends IButtonProps {
     onRenderOffIcon?: IRenderFunction<IButtonProps>;
     onRenderOnIcon?: IRenderFunction<IButtonProps>;
     showLabel?: boolean;
-    strings?: ControlBarButtonStrings;
+    strings?: Partial<ControlBarButtonStrings>;
     styles?: ControlBarButtonStyles;
     tooltipId?: string;
 }
 
 // @public
 export interface ControlBarButtonStrings {
-    label?: string;
-    offLabel?: string;
-    onLabel?: string;
-    tooltipContent?: string;
-    tooltipDisabledContent?: string;
-    tooltipOffContent?: string;
-    tooltipOnContent?: string;
+    label: string;
```

Comment:
This looks like a breaking change to the stable API (going from optional -> required is breaking)